nautobot/dcim/models/device_components.py

```diff
@@ -1019,7 +1037,7 @@ class InventoryItem(MPTTModel, ComponentModel):
     ]
 
     class Meta:
-        ordering = ("device__id", "parent__id", "_name")
+        ordering = ("_name",)
```

Comment:
django-tree-queries doesn't appear happy when using a compound `ordering` definition, and ordering by IDs (PKs) doesn't make sense for UUID PKs anyway.